client/src/panels/panel_declarations/finances/transfer_payments/tp_by_region.yaml

```diff
@@ -6,39 +6,29 @@ tp_by_region_title:
-    {{/if}}
-
-    Hover over or tap a province or territory in the map below to see the total transfer payments over the last five years. Note that the scale may change to accommodate the data displayed.
+    In **{{pa_last_year}}**, **{{subj_name subject}}** paid out **{{fmt_compact1_written subject_total_value}}** in {{gl_tt "transfer payments" "TP"}}. **{{fmt_compact1_written largest_total_value}}** ({{fmt_percentage1 largest_total_percent}}) went to recipients in **{{largest_total_prov}}**, making it the region receiving the largest total share of {{subj_name subject}}'s transfer payments.  
 
-  fr: |
-    #TODO
-    {{#if show_per_capita}}  
-    En **{{pa_last_year}}**, **{{subj_name subject}}** a versé **{{total_sum}}** en paiements de transfert. Les provinces bénéficiaires des paiements de transfert au-dessus 100 000 $ sont présentés ci-dessous, un montant divisé par la population de la région sélectionnée. Les bénéficiaires de **{{largest_prov}}** ont reçu la plus grande part divisée de **{{largest_per_capita}}** par habitant, soit un total de **{{largest_amount}}**.
-    {{else}}
-    En **{{pa_last_year}}**, **{{subj_name subject}}** a versé **{{total_sum}}** en paiements de transfert. Les provinces bénéficiaires des paiements de transfert au-dessus 100 000 $ sont présentés ci-dessous. Les bénéficiaires de **{{largest_prov}}** ont reçu la plus grande part totale de **{{largest_amount}}** ({{percent_of_total}}).
+    {{#if show_per_capita_data}}
+      When accounting for population statistics, the region with the largest per capita transfer payment value in **{{pa_last_year}}** was **{{largest_per_capita_prov}}**, at **{{fmt_compact1_written largest_per_capita_value}} per capita**.
+      {{#if compare_per_capita_to_largest_total}}
+        For comparison, the region receiving the largest total value, **{{largest_total_prov}}**, received **{{fmt_compact1_written largest_total_per_capita_value}} per capita**.
+      {{/if}}
     {{/if}}
-    
-    Survolez ou tapez sur une province ou un territoire dans la carte ci-dessous pour voir le total des paiements de transfert au cours des cinq dernières années. Notez que l'échelle peut changer pour s'adapter aux données affichées.
+  fr: |
+    TODO
 
-show_tp:
-  en: Transfer Payments
-  fr: Les Paiements de Transfert
 
-show_tp_per_capita:
-  en: Tranfer Payments per Capita
-  fr: Les Paiements de Transfert par Habitant
+tp_by_region_graph_usage:
+  en: Hover over or tap a province or territory in the map below to see the total transfer payments over the last five years. Note that the scale may change to accommodate the data displayed.
+  fr: Survolez ou tapez sur une province ou un territoire dans la carte ci-dessous pour voir le total des paiements de transfert au cours des cinq dernières années. Notez que l'échelle peut changer pour s'adapter aux données affichées.
```

Comment:
Split this out so it'd be easier to hide in the a11y mode, where it refers to a graph that doesn't exist.

Reply:
Split yaml text for clarity
👍